.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+#### Issue
+* PP-XXXX
+
+#### Description
+* A few sentences describing the overall goals of the pull request's commits.
+
+#### Steps to Test or Reproduce
```

Comment:
I thought this information was supposed to be present in the OpenJIRA ticket. Will QA need to reference both the ticket and the pull request? Will this information automatically be added to the ticket?